### PR TITLE
Bump bounds, move to newer stack LTS

### PR DESCRIPTION
**Pull request by michaelpj**
This bumps some bounds so that this package will build with the latest LTS snapshot.

I also bumped the resolver in the `stack.yaml` to match this.

The tests pass for me with this commit, so it doesn't seem to break anything.

### CODE REVIEW

**Review by jpvillaisaza — APPROVED** [review]
@michaelpj Thanks!

It looks like CI failed, I'll see if I can figure that out before merging.

**Comment by michaelpj** [comment]
Hm yeah, not sure what it's doing, it seems to be looking for something from the fpco website that isn't there...

**Comment by jpvillaisaza** [comment]
It worked fine after upgrading the Docker image as well. Thanks!

**Comment by jpvillaisaza** [comment]
@michaelpj Do you need a new version uploaded to Hackage with the changes here? I don't remember all the changes pending to be released and I'm not currently using this library, so just wanted to know whether you can wait on a new version being released or whether you're interested in becoming a collaborator (GitHub)/maintainer (Hackage). Thanks!

**Comment by michaelpj** [comment]
It would be great to get it on Hackage! I'm actually not sure what we use it for, cc @shmish111 what do you think?

**Comment by shmish111** [comment]
Actually we don't use it any more.

**Pull request merged**
